migrations/2018-06-22-00-standardize-request-invite-from-field.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Some members had an inconsistently named request_invite_from_uid field.
+ * The field was instead named requested_invite_from_uid. This standardizes
+ * the name on all members to the former.
+ */
+
+import * as path from "path";
+
+import * as admin from "firebase-admin";
+import {
+  Firestore,
+  QueryDocumentSnapshot,
+  DocumentSnapshot
+} from "@google-cloud/firestore";
+
+import { getDb } from "./helpers";
+import { request } from "http";
+
+async function standardizeRequestInviteFromField(db: Firestore) {
+  const members = (await db.collection("members").get()).docs;
+
+  for (const i in members) {
+    const member = members[i];
+
+    const requestInviteFromMemberId = member.get("request_invite_from_uid");
+    if (!requestInviteFromMemberId) {
+      const BAD_KEY_requestedInviteFromMemberId = member.get(
+        "requested_invite_from_uid"
+      );
+      if (BAD_KEY_requestedInviteFromMemberId) {
+        console.log(
+          `Renaming field for member with id ${
+            member.id
+          } from requested_invite_from_uid to request_invite_from_uid`
```

Comment:
Can we make it `request_invite_from_member_id`? I want to stop using `uid` everywhere except for when it's specifically referring to Firebase functionality.

Reply:
👍